Code/Tools/SceneAPI/SceneData/Tests/GraphData/GraphDataBehaviorTests.cpp

```diff
@@ -228,7 +228,7 @@ namespace AZ
                 }
             };
 
-            class GrapDatahBehaviorScriptTest
+            class GraphDataBehaviorScriptTest
```

Comment:
Whoops good catch on these :)

Reply:
sadly, the original author of this goof was me :)